.github/workflows/release.yml

```diff
@@ -0,0 +1,70 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+      - next
+
+jobs:
+  semantic-release:
+    name: Build & Publish
+    if: "!contains(github.event.head_commit.message, 'SKIP CI')"
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [10.x]
+    steps:
+
+      # Setup Git Repository & Node
+      #
+      - name: Checkout branch (${{ github.ref }})
+        uses: actions/checkout@master
+      - name: Configure git
+        run: |
+          git config user.name "Jon Ursenbach"
+          git config user.email "jon@readme.io"
+      - name: Setup node (v${{ matrix.node-version }})
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install packages
+        run: |
+          npm ci
+          npx lerna bootstrap --concurrency 2
+        env:
+          CI: true
+
+      # Version, build, tag, and publish the new release;
+      #
+      - name: Build new release
+        run: npm run release # see: .releaserc for config
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_ADMIN || secrets.GITHUB_TOKEN }} # auth to push release to remote repo
```

Comment:
@rafegoldberg Where should I get these Github and NPM tokens from?

Reply:
You only really the need the `GITHUB_TOKEN` one. I used a personal access token, but in the long run we should probably generate dedicated tokens for this. (I think an org-level admin can generate access tokens tied to the organization, rather than an individual account.)